### ISSUE TITLE
TypeAliases type unknown when verifying type completeness 

### ISSUE THREAD

**Issue by bijij** [microsoft/pyright#2261]
**Describe the bug**
when a `TypeAlias` exposed the `--verifytypes` option will consider the alias as untyped.

**To Reproduce**
Consider a directory strucure as follows
 ```
 /
   / test
   -  __init__.py
   - py.typed 
```

with `__init__.py` containting the following.
```py
from typing import TypeAlias

a: TypeAlias = int
```

`pyright --verifytypes test`

```
Package name: "test"
Package directory: "test"
Path of py.typed file: "test\py.typed"

Public modules: 1
   test

Symbols used in public interface:
test.a
  test\__init__.py:3:1 - error: Type unknown for variable "test.a"

Symbols exported by "test": 1
  With known type: 0
  With partially unknown type: 1
  Functions without docstring: 0
  Functions without default param: 0
  Classes without docstring: 0

Other symbols referenced but not exported by "test": 0
  With known type: 0
  With partially unknown type: 0

Type completeness score: 0%

Completed in 0.397sec
```

**Expected behavior**
I would expect the code to be verified as type-complete, Likely this would mean the completeness check ignores type aliases.

**Screenshots or Code**
![image](https://user-images.githubusercontent.com/8677174/131993480-6a090541-9ad1-42e7-90ff-5ac3a3ff8d5b.png)

**VS Code extension or command-line**
pyright 1.1.165

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

I'm really happy to hear that the `--verifytypes` option is getting used!

**Comment by erictraut** [comment]
This is included in pyright 1.1.166, which I just published. It will also be included in the next release of pylance.
